upup/pkg/kutil/delete_cluster_vsphere.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kutil
+
+import (
+	"fmt"
+)
+
+func (c *AwsCluster) listResourcesVSphere() (map[string]*ResourceTracker, error) {
```

Comment:
Yeah we can have it this way and fix it when they update the upstream. We will see the conflicts when we merge if they fixed it by that time.